tests/test_alter.py

```diff
@@ -0,0 +1,108 @@
+import pytest
+import re
+import datajoint as dj
+from . import schema as schema_any_module, schema_alter as schema_alter_module, PREFIX
+from .schema_alter import Parent, Experiment
+
+COMBINED_CONTEXT = {
+    **schema_any_module.LOCALS_ANY,
+    **schema_alter_module.LOCALS_ALTER,
+}
+
+
+@pytest.fixture
+def schema_alter(connection_test):
+    schema_any = dj.Schema(
+        PREFIX + "_alter",
+        context=schema_any_module.LOCALS_ANY,
+        connection=connection_test,
+    )
+    schema_any(schema_any_module.TTest)
+    schema_any(schema_any_module.TTest2)
+    schema_any(schema_any_module.TTest3)
+    schema_any(schema_any_module.NullableNumbers)
+    schema_any(schema_any_module.TTestExtra)
+    schema_any(schema_any_module.TTestNoExtra)
+    schema_any(schema_any_module.Auto)
+    schema_any(schema_any_module.User)
+    schema_any(schema_any_module.Subject)
+    schema_any(schema_any_module.Language)
+    schema_any(schema_any_module.Experiment)
+    schema_any(schema_any_module.Trial)
+    schema_any(schema_any_module.Ephys)
+    schema_any(schema_any_module.Image)
+    schema_any(schema_any_module.UberTrash)
+    schema_any(schema_any_module.UnterTrash)
+    schema_any(schema_any_module.SimpleSource)
+    schema_any(schema_any_module.SigIntTable)
+    schema_any(schema_any_module.SigTermTable)
+    schema_any(schema_any_module.DjExceptionName)
+    schema_any(schema_any_module.ErrorClass)
+    schema_any(schema_any_module.DecimalPrimaryKey)
+    schema_any(schema_any_module.IndexRich)
+    schema_any(schema_any_module.ThingA)
+    schema_any(schema_any_module.ThingB)
+    schema_any(schema_any_module.ThingC)
+    schema_any(schema_any_module.Parent)
+    schema_any(schema_any_module.Child)
+    schema_any(schema_any_module.ComplexParent)
+    schema_any(schema_any_module.ComplexChild)
+    schema_any(schema_any_module.SubjectA)
+    schema_any(schema_any_module.SessionA)
+    schema_any(schema_any_module.SessionStatusA)
+    schema_any(schema_any_module.SessionDateA)
+    schema_any(schema_any_module.Stimulus)
+    schema_any(schema_any_module.Longblob)
+
+    # Add nodes from schema_alter_module
+    schema_any(Experiment, context=schema_alter_module.LOCALS_ALTER)
+    schema_any(Parent, context=schema_alter_module.LOCALS_ALTER)
+
+    yield schema_any
+    schema_any.drop()
```

Comment:
```suggestion
@pytest.fixture
def schema_alter(connection_test, schema_any):
    schema_any(Experiment, context=schema_alter_module.LOCALS_ALTER)
    schema_any(Parent, context=schema_alter_module.LOCALS_ALTER)

    yield schema_any
    schema_any.drop()
```
This seems like a simpler solution unless there's a reason to keep it as is.